services/ingestion/poller/GoogleEventPoller/src/GoogleEventPoller/google_event_poller.py

```diff
@@ -38,12 +38,12 @@ def poll():
 	for calendar_id in calendar_ids_from_ssm:
 		events['data'].update(get_events(credsentials_from_ssm, calendar_id))
 
+	if bool(events['data']):
```

Comment:
Hvis du bare skal sjekke om det finnes noe i events['data'], trenger du ikke bool(). 
Hvis noe som helst er i events['data'] vil den bli True.
```suggestion
	if events['data']:
```